gapic/src/main/com/google/gapid/perfetto/views/TrackSelector.java

```diff
@@ -61,8 +61,6 @@ public TrackSelector(Composite parent, State.ForSystemTrace state, Theme theme)
     tree = createTree(container, theme);
```

Comment:
nit: Group in line 55 is superfluous.

(I couldn't figure out how to add comment on an unchanged line)